releasenotes/notes/transpiler-control-flow-708896bfdb51961d.yaml

```diff
@@ -0,0 +1,78 @@
+---
+features:
+  - |
+    Control-flow operations are now supported through the transpiler at
+    optimization levels 0 and 1.  One can now construct a circuit such as
```

Comment:
I might refer to the `transpile()` function here, because the optimization_level argument is on `transpile()`.

Reply:
Done in a53a4a3fc.